src/dos/dos_mscdex.cpp

```diff
@@ -1093,9 +1103,30 @@ static Bit16u MSCDEX_IOCTL_Optput(PhysPt buffer,Bit8u drive_unit) {
 
 static MountType MSCDEX_GetMountType(const char *path)
 {
+	const char *cdName;
+	char buffer[512];
+	strcpy(buffer, path);
+#if defined(WIN32)
+	upcase(buffer);
+#endif
+
+	int num = SDL_CDNumDrives();
+	// If cd drive is forced then check if its in range and return 0
+	if ((forceCD >= 0) && (forceCD < num)) {
+		LOG(LOG_ALL, LOG_ERROR)("CDROM: Using drive %d", forceCD);
+		return MountType::PHYSICAL;
+	}
+
+	// compare names
+	for (int i = 0; i < num; i++) {
+		cdName = SDL_CDName(i);
+		if (strcmp(buffer, cdName) == 0)
```

Comment:
Same as earlier regarding `std::string` and `==`

Reply:
Done.